src/app/docs/lookup/lookup-docs.component.html

```diff
@@ -69,10 +70,112 @@
+        <sky-docs-design-guideline-thumbnail
+            *skyThemeIf="'modern'" caption="Do use lookup fields when users may need to add options to the list."
+                                   captionType="success"
+                                   imageSource="~/assets/img/guidelines/lookup/lookup-usage-3-modern.png">
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+        
+      <sky-docs-design-guideline
+        heading="Don't use when"
+        headingStyle="danger"
+      >
+        <p>
+            Don’t use lookup fields when short lists include 5 options or fewer, especially if it is valuable for users to see all options at once. Use
+            <a *skyThemeIf="'default'" href="https://developer.blackbaud.com/skyux-forms/docs/checkbox">checkboxes</a>
+            <a *skyThemeIf="'modern'" href="https://developer.blackbaud.com/skyux-forms/docs/checkbox?svcid=skyux-modern">checkboxes</a> or 
+            <a *skyThemeIf="'default'" href="https://developer.blackbaud.com/skyux-forms/docs/radio">radio buttons</a>
+            <a *skyThemeIf="'modern'" href="https://developer.blackbaud.com/skyux-forms/docs/radio?svcid=skyux-modern">radio buttons</a> instead.
+        </p>
+        <sky-docs-design-guideline-thumbnail
+            *skyThemeIf="'default'" caption="Don't use lookup fields when users choose options from a short list."
+                                    captionType="danger"
+                                    imageSource="~/assets/img/guidelines/lookup/lookup-usage-4-default.png">
+        </sky-docs-design-guideline-thumbnail>
+        <sky-docs-design-guideline-thumbnail
+            *skyThemeIf="'modern'" caption="Don't use lookup fields when users choose options from a short list."
+                                   captionType="danger"
+                                   imageSource="~/assets/img/guidelines/lookup/lookup-usage-4-modern.png">
+        </sky-docs-design-guideline-thumbnail>
+        <sky-docs-design-guideline-thumbnail
+            *skyThemeIf="'default'" caption="Do use different selection inputs for a short list of options."
+                                    captionType="success"
+                                    imageSource="~/assets/img/guidelines/lookup/lookup-usage-5-default.png">
+        </sky-docs-design-guideline-thumbnail>
+        <sky-docs-design-guideline-thumbnail
+            *skyThemeIf="'modern'" caption="Do use different selection inputs for a short list of options."
+                                   captionType="success"
+                                   imageSource="~/assets/img/guidelines/lookup/lookup-usage-5-modern.png">
+        </sky-docs-design-guideline-thumbnail>
+      </sky-docs-design-guideline>
+        
+      <sky-docs-design-guideline>
+        <p>
+            Don’t use lookup fields when options are highly predictable, especially if they are not unique to the system. Use HTML select fields instead.
```

Comment:
```suggestion
            Don’t use lookup fields when options are highly predictable, especially if they are not unique to the system. Use HTML select fields instead.
            Don’t use lookup fields when options are highly predictable or when users can easily browse options in an expected order. Use HTML select fields instead. For example, while a list of states is long, it’s predictable and most users are familiar with the options, so the native HTML select field is the preferable option.
```
This rewrite is based on our conversation about the component and the need to emphasize the use case for lookup fields vs. HTML select fields. We talked about better emphasizing the difference in the use cases, so hopefully I captured what we discussed here and in the edits to the first "Use when" item. But we can revise as necessary.